src/button.test.visuals.ts

```diff
@@ -3,7 +3,7 @@ import { expect, test } from '@playwright/test';
 test.describe('Button', () => {
   test.describe('Light Mode', () => {
     test.describe('disabled', () => {
-      test('variant="primary"', async ({ page }, test) => {
```

Comment:
This way of naming tests doesn't play out so well with boolean attributes [when](https://github.com/CrowdStrike/glide-core/pull/743/files#diff-c4014b8cb449a0466d2fd86d6d3ccdd37a6599b94a25126cc82ea1fefc43bec3R26-R46) we need to test both `true` and `false`.

As a test name, `open` makes sense but `open=false` doesn't because that's not how the attribute is used in practice. And `open=${false}` is a little unwieldy.